### DISCUSSION TITLE
Installation of the old version (version 3.00)

### DISCUSSION THREAD

**Discussion by shaheidula** [mdolab/dafoam#412] [Installation]
Dear Prof. Ping He, 
I would like to install an old version (version 3.0.0) as our self-developed turbulence model can only work on it. However, as the lack of dependency details, we are struggling to install it. Could you please help us with this?

**Comment by friedenhe** [comment]
3.0.0 was released on April 2022, so you can go to the DAFoam.github.io commit around that time, like this one https://github.com/DAFoam/DAFoam.github.io/blob/bc9149f19ad1dd9d2142f4887465769a71c15d6e/pages/mydoc/mydoc_installation_source.md
